networkx/algorithms/assortativity/correlation.py

```diff
@@ -150,12 +149,11 @@ def degree_pearson_correlation_coefficient(G, x="out", y="in", weight=None, node
     .. [2] Foster, J.G., Foster, D.V., Grassberger, P. & Paczuski, M.
        Edge direction and the structure of networks, PNAS 107, 10815-20 (2010).
     """
-    import scipy as sp
-    import scipy.stats  # call as sp.stats
+    import scipy.stats
```

Comment:
Let's keep using this import pattern.  It was put in place to prepare for upcoming changes using lazy loading.